app/endpoints/users.py

```diff
@@ -879,6 +879,9 @@ async def read_user_profile_picture(
     Unauthenticated users can use this endpoint (needed for some OIDC services)
     """
 
+    # user_id should not contains wildcards, which could interpreted by glob.glob (https://docs.python.org/fr/3.10/library/glob.html)
+    user_id = user_id.replace("*", "")
```

Comment:
We could maybe check user_id is a valid UUID

Reply:
You're right that's far better